guide/02-api-overview/overview241.ipynb

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
<p>Typo in the 3rd sentence: uploading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Thanks! Fixed it.</p>